### DISCUSSION TITLE
How switch to GPT-4 model?

### DISCUSSION THREAD

**Discussion by pandability** [n3d1117/chatgpt-telegram-bot#112] [Q&A] [answered]
Hi, I saw that in last release 0.2.1 was added GPT4 model support: 
gpt-4 model support by [@carlsverre](https://github.com/carlsverre) in [#91](https://github.com/n3d1117/chatgpt-telegram-bot/pull/91)

How I can enable it?

**Comment by n3d1117** [comment, marked as answer]
Hi @pandability. If you have early access to gpt-4 APIs, all you need to do is add this to your .env:

```
OPENAI_MODEL="gpt-4" # or "gpt-4-0314" or "gpt-4-32k" or "gpt-4-32k-0314"
```

and it should work!

**Comment by AlexKast37** [comment]
As I understand, you need include these models to "openai_helper.py" line 15-16, but not to .env, because in .env is only info about telegram token and api openai and admin user?

**Reply by n3d1117** [reply]
No, you can add it to env. All configuration options should go to .env
